level1/bmi.rb

```diff
@@ -0,0 +1,18 @@
+class Bmi
+
+  @@cant_decimals = 1
+  def self.calculate(weigth, size)
+    if size<1 or weigth<1
```

Comment:
`AND` and `OR` have lower precedence, use `&&` and `||`
```suggestion
    if size < 1 || weigth < 1
```